backend/YouTubeV2.Application/Services/AzureServices/BlobServices/BlobImageService.cs

```diff
@@ -37,5 +40,15 @@ private async Task UploadImageAsync(byte[] bytes, string fileName, string blobCo
                 new BlobUploadOptions { HttpHeaders = new BlobHttpHeaders { ContentType = "image/png" } },
                 cancellationToken);
         }
+
+        private async Task DeleteImageAsync(string fileName, string blobContainerName, CancellationToken cancellationToken = default)
```

Comment:
idk if thats necessary. I think that uploading an image with the same name will overwrite the old image. You can check if thats true

Reply:
we still should delete when deleting a user. Or do you think about uploading an empty image when deleting?